### PR TITLE
Update evoker_devastation.yaml

### PR DESCRIPTION
**Pull request by Pandaabear0**
Added M+ and Cleave build.
Made a second proposed change since I missed to remove a space on the first one. Sorry about that.

### CODE REVIEW

**Comment by Pandaabear0** [comment]
I'm a bit new to github and how all of this works, so let me know if there is something I missed or what I need to do in order to complete this pull.

File: bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml [hunk 1]
```diff
@@ -4,3 +4,7 @@
 #   - spec_talents=talent_name_d:1/talent_name_e:0/talent_name_f:3
 # Cool Other Build:
 #   - talents=WOW_EXPORT_STRING
+M+ Build:
+   - talents=BsbBnWR3R1Z1HmbUKIpvm3XXhTDCAAAAAAKaIJikEJkEJlkk0CSBJJhkEIJJSESSC
+Cleave Build:
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKapIJikEJIRSLJJNi0CJSIJSQSiEhkkA
```
Comment (Bloodmallet) [inline bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml]: Talked with Preheat and got this talent string for a Cleave build instead: BsbBAAAAAAAAAAAAAAAAAAAAAAkAAAAAAgiWKSiIJkQSkUSS0gUCJSIJEkkIRikkA

could you update your PR using this one?

File: bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml [hunk 2]
```diff
@@ -4,3 +4,7 @@
 #   - spec_talents=talent_name_d:1/talent_name_e:0/talent_name_f:3
 # Cool Other Build:
 #   - talents=WOW_EXPORT_STRING
+M+ Build:
+   - talents=BsbBnWR3R1Z1HmbUKIpvm3XXhTDCAAAAAAKaIJikEJkEJlkk0CSBJJhkEIJJSESSC
```
Comment (Bloodmallet) [inline bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml:8]: 👍

**Review by Bloodmallet — CHANGES_REQUESTED** [review]
Format works. I tried your changes locally and it worked fine. Thank you for the PR! Please have a look at the other comments.

**Comment by Pandaabear0** [comment]
So I just went to bed, so I had to do this on phone. Didn't wanna leave you waiting. I think I updated it. If not, I will do it on PC when I wake.

**Review by Bloodmallet — APPROVED** [review]
Thank you very much! Devastation Evokers will have these talent paths in their next data regeneration.

**Pull request merged**